web_responsive/README.rst

```diff
@@ -0,0 +1,123 @@
+
+Usage
+=====
+
+The following keyboard shortcuts are implemented:
+
+* Toggle App Drawer - `ActionKey <https://en.wikipedia.org/wiki/Access_key#Access_in_different_browsers>` + ``A``
+* Navigate Apps Drawer - Arrow Keys
+* Type to select App Links
+* ``esc`` to close App Drawer
+
+Known issues / Roadmap
+======================
+
+Note: Data added to the footer ``support_branding`` is not shown while using
+this module.
+
+* Provide full menu search feature instead of just App search
+* Drag drawer from left to open in mobile
+* Figure out how to test focus on hidden elements for keyboard nav tests
+* If you resize the window, body gets a wrong ``overflow: auto`` css property
+  and you need to refresh your view or open/close the app drawer to fix that.
+* Override LESS styling to allow for responsive widget layouts
+* Adding ``oe_main_menu_navbar`` ID to the top navigation bar triggers some
+  great styles, but also `JavaScript that causes issues on mobile
+  <https://github.com/OCA/web/pull/446#issuecomment-254827880>`_
+* Sticky header and footer in list view only works on certain browsers:
+  https://caniuse.com/#search=sticky (note that the used feature is in
+  `thead`).
+* On Android (FireFox) - clicking the search icon does not
+  focus the search input.
+* On Android (FireFox & Chrome) - clicking the search query input will
+  show the on screen keyboard for a split second, but the App Drawer
+  immediately closes and the keyboard closes with it.
+* Filter menu items completely on client-side, to make it smoother and allow
+  users to filter on complete paths of menus and not only on the last item.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_responsive%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* LasLabs
+* Tecnativa
+* Alexandre Díaz
+
+Contributors
+~~~~~~~~~~~~
+
+* Dave Lasley <dave@laslabs.com>
+* Jairo Llopis <jairo.llopis@tecnativa.com>
+* Dennis Sluijk <d.sluijk@onestein.nl>
+* Sergio Teruel <sergio.teruel@tecnativa.com>
+* Alexandre Díaz <dev@redneboa.es>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/web <https://github.com/OCA/web/tree/11.0/web_responsive>`_ project on GitHub.
```

Comment:
@Yajo IMO Change target branch `12.0`

Reply:
The readme will be rebuilt automatically after merge by the bot, don't worry about these things.

FTR, if you see a `readme/` folder in an addon, that's the important part.